civcraftConfigs/zionconfig.yml

```diff
@@ -22,31 +25,43 @@ daytime_modifier:
       biomes:
        - GLOBAL
 
+fireball:
+ netherFireRain:
+ frequency: 1h
```

Comment:
lol we thought the fire rain was more like a rainstorm and it happend once an hr we didnt realize it was only one fireball per hr ill change that in a bit!